README.md

```diff
@@ -25,23 +25,23 @@ As a fully-managed database service, Cloud SQL supports multiple major versions
 It’s particularly useful for organizations that want to avoid extended support fees associated with Cloud SQL extended support for MySQL and PostgreSQL end-of-life versions, or that want to take advantage of the performance and availability features of Enterprise Plus editions.
 
 
-Key features 
+## Key features 
 The Cloud SQL upgrade tool provides: 
 
-Automated pre-upgrade assessment, where checks are curated based on recommendations available for MySQL and PostgreSQL upgrades, as well as from insights from real customer experiences.
+* Automated pre-upgrade assessment, where checks are curated based on recommendations available for MySQL and PostgreSQL upgrades, as well as from insights from real customer experiences.
 
-Detailed assessment reports
+* Detailed assessment reports
 
-Automated scripts to resolve issues
+* Automated scripts to resolve issues
 
-In-place major version and Enterprise Plus upgrades leveraging Cloud SQL’s in-place major version upgrade feature.
+* In-place major version and Enterprise Plus upgrades leveraging Cloud SQL’s in-place major version upgrade feature.
 
-You can visualize how these elements work together in the diagram below.
+* You can visualize how these elements work together in the diagram below.
 
 https://storage.googleapis.com/gweb-cloudblog-publish/images/1-_Architecture_Rz3TUpk.max-1600x1600.png
 
 
-Using the Cloud SQL upgrade tool 
+## Using the Cloud SQL upgrade tool 
```

Comment:
you can remove the last line - 
If you are interested in the tool,  reach out to your Google representative. and add 1st line "Clone the repo and navigate the /path to locate the binary file